noxfile.py

```diff
@@ -2,7 +2,7 @@
 from nox import session
 
 PYTHON_VERSIONS = ["3.8", "3.9", "3.10"]
-SPHINX_VERSIONS = ["5.0"]
+SPHINX_VERSIONS = ["5.0", "6.2.1", "7.1.2", "7.2.5"]
```

Comment:
For Sphinx-Needs we have this matrix settings:
![image](https://github.com/useblocks/sphinx-test-reports/assets/998700/b7580660-1b75-42af-bdc2-a91b47fa5b56)

Should we use the same versions here as well?

Are there any important changes between `7.1.2` and `7.2.5` which will let our tests behave differently?

Reply:
First of all: i would be happy if all useblocks  provided sphinx extensions could match the same build/testing environment, this would make integration into my used building/testing environment

I choosed 7.1.2 cause this was the last version before the deprecation (introduced with 7.2.0)

It seems to me that with 7.2.5 the App object required by the spinx tests api changed the interface from PathLike Objects to pathlib.Paths. So testing one version AFTER this change and one before seems useful.